scripts/apps/vocabularies/components/ItemsTableComponent.tsx

```diff
@@ -52,7 +52,7 @@ export default class ItemsTableComponent extends React.Component<any, any> {
     }
 
     setCaretPosition(ctrl, pos) {
-        if (ctrl.setSelectionRange) {
+        if (ctrl.setSelectionRange && pos !== null) {
```

Comment:
Use `!=` nstead of `!==` when comparing with `null` to also catch undefined. What is pos and why was it `null`?

Reply:
`pos` is cursor position so it is null in case of color.

Reply:
How is setting caret position for a table related with color picker? It looks to me that the function shouldn't be called in the first place.